isis/src/base/objs/Application/Application.h

```diff
@@ -31,6 +31,7 @@ namespace Isis {
   class Gui;
   class IException;
   class Progress;
+  class PvlObject; 
```

Comment:
Is this needed?

Reply:
left over from the class approach, removed it